[2024-05-21T22:09:54.982+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-21T22:09:55.129+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: kaggle_dataset.transformacion manual__2024-05-21T22:05:48.748499+00:00 [queued]>
[2024-05-21T22:09:55.159+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: kaggle_dataset.transformacion manual__2024-05-21T22:05:48.748499+00:00 [queued]>
[2024-05-21T22:09:55.161+0000] {taskinstance.py:2306} INFO - Starting attempt 2 of 6
[2024-05-21T22:09:55.640+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): transformacion> on 2024-05-21 22:05:48.748499+00:00
[2024-05-21T22:09:55.672+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=333) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-05-21T22:09:55.678+0000] {standard_task_runner.py:63} INFO - Started process 357 to run task
[2024-05-21T22:09:55.681+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'kaggle_dataset', 'transformacion', 'manual__2024-05-21T22:05:48.748499+00:00', '--job-id', '18', '--raw', '--subdir', 'DAGS_FOLDER/kaggle_dataset.py', '--cfg-path', '/tmp/tmp3c3vk15j']
[2024-05-21T22:09:55.694+0000] {standard_task_runner.py:91} INFO - Job 18: Subtask transformacion
[2024-05-21T22:09:55.924+0000] {task_command.py:426} INFO - Running <TaskInstance: kaggle_dataset.transformacion manual__2024-05-21T22:05:48.748499+00:00 [running]> on host 36dd2f748723
[2024-05-21T22:09:56.254+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='kaggle_dataset' AIRFLOW_CTX_TASK_ID='transformacion' AIRFLOW_CTX_EXECUTION_DATE='2024-05-21T22:05:48.748499+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-05-21T22:05:48.748499+00:00'
[2024-05-21T22:09:56.258+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-21T22:09:56.313+0000] {logging_mixin.py:188} INFO - Hola Mundo
[2024-05-21T22:09:56.315+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-05-21T22:09:56.319+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-21T22:09:56.352+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=kaggle_dataset, task_id=transformacion, run_id=manual__2024-05-21T22:05:48.748499+00:00, execution_date=20240521T220548, start_date=20240521T220955, end_date=20240521T220956
[2024-05-21T22:09:56.435+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-21T22:09:56.540+0000] {taskinstance.py:3498} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-05-21T22:09:56.547+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
